.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[Makefile]
+indent_style = tab
```

Comment:
We currently use spaces for makefiles as well. At least, my script did nothing to them except trimming trailing spaces..

Reply:
? At least some versions of `make` infer a semantic distinction between tab indent and space indent.

Reply:
Ah, I've been thinking about `cmake`.